packages/cms-data-sync/src/utils/rich-text.ts

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable no-console */
+/* eslint-disable no-param-reassign */
+
+import { Document } from '@contentful/rich-text-types';
+import { parseHtml } from 'contentful-html-rich-text-converter';
+
+export const clearParsedHtmlOutput = (htmlDocument: Document) => ({
+  ...htmlDocument,
+  content: htmlDocument?.content
+    .map((node) =>
+      node.nodeType === 'unordered-list' || node.nodeType === 'ordered-list'
```

Comment:
you may want to add a comment about what we are doing here

Reply:
added in here https://github.com/yldio/asap-hub/pull/2290/commits/7c70e7be9e43e274abd5a374ec74d9194c9ba040